installer/datafiles/Base_SCXCore.data

```diff
@@ -123,6 +125,14 @@ DeleteSoftLinkToSudo() {
     fi
 }
 
+%DeleteSoftLinkToTmpDir
+DeleteSoftLinkToTmpDir() {
+    if [ -L ${{TMP_LOC}} ]; then
```

Comment:
Hmph. I didn't think that -L worked everywhere in 'test' (what you're using here in Bash), but given that the same code is just above for Sudo, I guess it does?

I presume you did test this properly in the installer, and all was fine?  If not, please do, just for sanity sake.